### DISCUSSION TITLE
Applying the same augmentation stack with different parameters to multiple images

### DISCUSSION THREAD

**Discussion by mikel-brostrom** [albumentations-team/albumentations#1405] [Q&A] [answered]
I am aware of this: https://albumentations.ai/docs/examples/example_multi_target/

But it is possible to apply the same stack but with different parameters to a set of images?
The idea here is to pass the differently augmented images to Mosaic.

**Comment by Dipet** [comment, marked as answer]
> But it is possible to apply the same stack but with different parameters to a set of images?

Call transforms twice
